src/components/work/Footer/Footer.tsx

```diff
@@ -69,9 +69,10 @@ const socialData = [
   },
 ];
 
-const Footer = () => (
+export const Footer = () => (
   <Switch>
     <Route path="/work/account" component={DetailedFooter} />
+    <Route exact path="/work/listings/:id" component={NoopComponent} />
```

Comment:
we can change `component=` to `render=` and pass down className as a prop for the component

Reply:
ty